OpenRA.Mods.Common/Traits/Render/SelectionDecorations.cs

```diff
@@ -33,17 +33,18 @@ public class SelectionDecorationsInfo : ITraitInfo, ISelectionDecorationsInfo
 
 		public readonly Color SelectionBoxColor = Color.White;
 
+		public readonly string Image = "pips";
+
 		public object Create(ActorInitializer init) { return new SelectionDecorations(init.Self, this); }
 
+		[Desc("Don't change the order of these sequences."), SequenceReference("Image")]
```

Comment:
In the long term (or possibly even sooner) this will be replaced with something similar to `WithDecoration` that allows for arbitrary placement and sprite choice for pips.